CHANGELOG.md

```diff
@@ -8,9 +8,12 @@ The changelog for `IGListKit`. Also see the [releases](https://github.com/instag
 This release closes the [2.1.0 milestone](https://github.com/Instagram/IGListKit/milestone/2).
 
 ### Enhancements
-
 - Added support for macOS. Note: this is *only* for the Diffing components. There is **no support** for `IGListAdapter`, `IGListSectionController`, and other components at this time. [Guilherme Rambo](https://github.com/insidegui) [(#235)](https://github.com/Instagram/IGListKit/pull/235)
 
+- Added ability to adjust Diffing behavior with `IGListDiffBehavior`, `IGListDiffWithBehavior`, and `IGListDiffPathsWithBehavior`.
+
+- Added algorithm for diffing with incremental moves which can be used with `IGListDiffBehaviorIncrementalMoves`.
```

Comment:
Make sure to follow the other changelog formats and include a link to your profile (if you want it) and this PR